src/widgets/helper/SearchPopup.cpp

```diff
@@ -212,6 +215,16 @@ bool SearchPopup::eventFilter(QObject *object, QEvent *event)
     return false;
 }
 
+void SearchPopup::themeChangedEvent()
+{
+    BasePopup::themeChangedEvent();
+
+    // NOTE: This currently overrides the QLineEdit's font size.
+    // If the dialog open is when the theme is changed, things will look a bit off.
```

Comment:
```suggestion
    // If the dialog is open when the theme is changed, things will look a bit off.
```

Reply:
Fixed in [`4df4cb3` (#6055)](https://github.com/Chatterino/chatterino2/pull/6055/commits/4df4cb3eaf756a175ba95c1e6bb781194e69fa2b) - thanks!